src/dhtproto/client/DhtClient.d

```diff
@@ -720,6 +724,68 @@ public class DhtClient : IClient
     mixin NeoSupport!();
 
 
+    /***************************************************************************
+
+        Constructor with support for the neo and legacy protocols, using various
+        default settings for ease of construction.
+
+        This ultra-minimal constructor is for convenience of scripts and tests.
+        It uses the following default settings:
+        * Using the task scheduler's epoll instance.
+        * If the task scheduler has not been initialised, this ctor initialises
+          it with its default configuration.
+        * For authentication, the name "test" and the init key are used.
+        * Unless a connection notifier is provided, a default notifier -- which
+          logs all events -- is used (see this.defaultConnNotifier).
+
+        Params:
+            conn_notifier = delegate which is called when a connection-related
+                event occurs on a neo connection. Of type:
+                void delegate ( Neo.DhtConnNotification )
+
+    ***************************************************************************/
+
+    public this ( Neo.DhtConnectionNotifier conn_notifier = null )
+    {
+        if ( !isSchedulerUsed() )
+            initScheduler(Scheduler.Configuration.init);
```

Comment:
Mihails said elsewhere that he doesn't think doing this is advisable. Waiting for his response on https://github.com/sociomantic-tsunami/ocean/pull/612#issuecomment-414235269.

Reply:
So, is this blocked or not :-)?